tests/acceptance/customCommands/uploadRemote.js

```diff
@@ -15,11 +15,11 @@ class UploadRemote extends events.EventEmitter {
     const buffers = []
     const zip = archiver('zip')
     /*
-    We add all the buffered data to  a list, and then,
-    we concat all the buffer instances, base64encode it and send to the remote
-    selenium browser. On completion, we get the result (containing the destination path to where the file
-    was uploaded) which we can retrieve from the callback
-     */
+            We add all the buffered data to  a list, and then,
```

Comment:
Doc block looks weird, indentation wise and I'd have expected a `*` in every line of the doc block.

Reply:
thanks, fixed